bats/tests/helpers/defaults.bash

```diff
@@ -40,3 +41,15 @@ using_image_allow_list() {
 using_windows_exe() {
     is_true "$RD_USE_WINDOWS_EXE"
 }
+
+using_networking_tunnel() {
+    is_true "$RD_USE_NETWORKING_TUNNEL"
+}
+
+if using_windows_exe && is_unix; then
```

Comment:
```suggestion
if using_windows_exe && !is_windows; then
```
That's more correct semantically (we won't support `using_windows_exe` on Amiga, for example.)  I realize that this ends up being the same thing right now…